gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageClientWriteChannel.java

```diff
@@ -45,7 +45,8 @@ class GoogleCloudStorageClientWriteChannel extends BaseAbstractGoogleAsyncWriteC
   private static final GoogleLogger logger = GoogleLogger.forEnclosingClass();
 
   private final StorageResourceId resourceId;
-  private WriteChannel writeChannel;
+  private WritableByteChannel writableByteChannel;
+  private final BlobWriteSession blobWriteSession;
```

Comment:
why we need this as an instance variable?

Reply:
https://github.com/GoogleCloudDataproc/hadoop-connectors/pull/1049#discussion_r1325676408

Reply:
Other than in the constructor, where is blobWriteSession getting used?

Reply:
It will be used to get the response of WrtiteObject.